agents/postgres/pgstatstatements/pgstatstatements.go

```diff
@@ -206,13 +207,14 @@ func makeBuckets(current, prev map[int64]*pgStatStatementsExtended, l *logrus.En
 			l.Debugf("Normal query: %s.", currentPSS)
 		}
 
+		fingerprint, _ := truncate.Query(currentPSS.Query)
```

Comment:
it's already truncated here https://github.com/percona/pmm-agent/blob/2c5ff8cc17b0416f770daa9cb37d99bc0e55f04a/agents/postgres/pgstatstatements/stat_statements_cache.go#L140